src/keywords.js

```diff
@@ -10,6 +10,13 @@ export syntax galat = function (ctx) {
     return #`false`;
 }
 
+
+// isNaN
+export syntax kya_galat = function (ctx) {
+    let params = ctx.next().value
+    return #`isNaN ${params}`;
+}
+
 // if
```

Comment:
How? 
The isNaN() function determines whether a value is NaN (Not-a-Number) or not
I chose "kya_galat" because it returns "true" if the given value is NaN; otherwise, "false."
Do you have any suggestions?